src/components/geonode.jsx

```diff
@@ -87,7 +87,16 @@ class GeoNodeViewer extends React.Component {
   }
   updateMap(props) {
     if (props.config) {
-      var tileServices = [];
+      var tileServices = [
+        {
+          name: 'Satellite',
+          description: 'ESRI world imagery',
+          endpoint: 'http://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/{z}/{y}/{x}',
+          standard: 'XYZ',
+          attribution: 'Tiles &copy; Esri &mdash; Source: Esri, i-cubed, USDA, USGS, AEX, GeoEye, Getmapping, Aerogrid, IGN, IGP, UPR-EGP, and the GIS User Community',
+          thumbnail: 'http://server.arcgisonline.com/ArcGIS/rest/services/World_Imagery/MapServer/tile/0/0/0'
```

Comment:
I don't think we should do it this way, this is normally managed on the Django side in geonode for the base maps. Imagine people in a disconnect environment.